examples/scripts/dpo.py

```diff
@@ -148,6 +148,7 @@ def split_prompt_and_responses(sample) -> Dict[str, str]:
         optim="rmsprop",
         warmup_steps=150,
         report_to=script_args.report_to,
+        bf16=True,
```

Comment:
This is needed for compat with the deepspeed configs - I think it's also fine to have as a default in any case